spec/requests/user_settings_spec.rb

```diff
@@ -58,6 +58,12 @@
       expect(response.body).to include("Username is too short")
     end
 
+    it "returns error if Profile image is too large" do
+      profile_image = fixture_file_upload("files/large_profile_img.jpg", "image/jpeg")
```

Comment:
Great job @bolariin, thanks!

Just a small small thing: could you make the image, just a little bit over the limit? This way it takes just a little bit less time to read an image a little bit over 2mb instead of one of 5 mb, since these tests are run continuously for each commit :)